### PR TITLE
Lighter block DOM: Button(s)

### PR DESCRIPTION
**Pull request by ellatrix**
## Description

Makes the Button and Buttons blocks light blocks. To do: fix styling.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by ellatrix** [comment]
Looks like we first need to replace the fallback styles with `useColors`. @jorgefilipecosta Does `useColors` currently support gradients?

File: packages/block-library/src/button/index.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ export const settings = {
 	supports: {
 		align: true,
```
Comment (youknowriad) [inline packages/block-library/src/button/index.js]: How do we deal with this :)

**Comment by jorgefilipecosta** [comment]
> Looks like we first need to replace the fallback styles with useColors. @jorgefilipecosta Does useColors currently support gradients?


Hi @ellatrix, useColors currently does not supports gradients but we have something equivalent for gradients "__experimentalUseGradient".

**Comment by ZebulanStanphill** [comment]
In case anyone was wondering, this was closed because the Button block and Buttons block were already updated to use a light block wrapper in #21266 and #21822 respectively.

**Pull request closed without merging**